extensions/adobe/experience/customerJourneyManagement/ajo-entity-mixins.schema.json

```diff
@@ -0,0 +1,195 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/ajo",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Profile - AJO Entities",
+  "description": "Denormalized AJO Entities.",
```

Comment:
A set of meta-data fields related to various AJO entities which are related to message sent to profile